handbook/engineering/hiring/software-engineer-security.md

```diff
@@ -28,13 +28,32 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 1. You write a short security assessment, taking less than **1hr** of your time, explaining a security vulnerability [sample problem](https://docs.google.com/document/d/1oXhjU_3y2uhpmWJ2hD0NTaqSanUAOlKDFFmjazAU3Rg/).
 1. We schedule remote interviews over video chat across multiple days.
    - 1h **Security architecture:** We give you an open problem statement and you walk us through how you would solve the problem.
+      - Will interview with 2 of the following:
+         - At least one member of the [security team](/handbook/engineering/security/index.md#members)
+         - Any engineer mentioned in this document.
    - 1h **Technical experience:** We ask you about your past work and accomplishments.
+      - Will interview with 2 of the following:
+        - [Stephen Gutekanst](/company/team/index.md#stephen-gutekanst)
+        - [Keegan Carruthers-Smith](/company/team/index.md#keegan-carruthers-smith)
+        - [Thorsten Ball](/company/team/index.md#thorsten-ball-he-him)
+        - [Eric Fritz](/company/team/index.md#eric-fritz-he-him)
+        - [Asdine El Hrychy](/company/team/index.md#asdine-el-hrychy)
+        - [Dax McDonald](/company/team/index.md#dax-mcdonald-he-him)
+        - [Elizabeth Stirling](/company/team/index.md#elizabeth-stirling-she-her)
    - 1h **Team collaboration:** We ask you about how you work and communicate in a team setting, and how you handle tricky situations.
+      - - Will interview with 2 of the following:
+         - [Christina Forney](../../../company/team/index.md#christina-forney-she-her)
+         - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
+         - [Eric Broody-Moore](../../../company/team/index.md#eric-brody-moore)
+         - [Pooja Jain](../../../company/team/index.md#pooja-jain-she-her)
+         - [Dan Adler](../../../company/team/index.md#dan-adler-he-him)
+         - If we have at least one of the above, the second interviewer can be [any engineer on the security team](/handbook/engineering/security/index.md#members)
+            or any engineer mentioned in this document.
    - 30m **VP Engineering**
    - 30m **CTO**
    - 30m **CEO**
-1. We check your references.
-1. We make you a job offer.
+2. We check your references.
+3. We make you a job offer.
```

Comment:
2 and 3 actually aren't correct since there are fours 1s above this. I would leave it as all ones.
```suggestion
1. We check your references.
1. We make you a job offer.
```